.devops/code-review-pipelines.yml

```diff
@@ -100,16 +100,64 @@ stages:
               echo "##[debug] Change placeholder for ecommerce-cdc-service set branch: $(BRANCH_NAME)"
               sed -i "s/^ECOMMERCE_CDC_COMMIT_SHA=.*$/ECOMMERCE_CDC_COMMIT_SHA=$(BRANCH_NAME)/" .env
             displayName: 'Change ecommerce-cdc-service branch name ecommerce-local .env'
-          - script: docker compose up pagopa-ecommerce-cdc-service mongo mongo-init redis -d
-            displayName: 'Run app with docker-compose'
-            continueOnError: true
+          - script: |
+              # Create temporary override file for CDC test configuration
+              cat > docker-compose.cdc-override.yml << 'EOF'
+              services:
+                pagopa-ecommerce-transactions-service:
+                  environment:
+                    - TRANSACTIONSVIEW_UPDATE_ENABLED=false
+                pagopa-ecommerce-event-dispatcher-service:
+                  environment:
+                    - TRANSACTIONSVIEW_UPDATE_ENABLED=false
+              EOF
```

Comment:
instead of performing overrides here can we just change services env files as we do for .env file (to change branch)?

I would do it as single script job so that this can be deleted altogether once switch to cdc is definitive